core/modules/user/js/user.js

```diff
@@ -143,20 +143,28 @@ Backdrop.behaviors.passwordConfirm = {
  *
  * Returns the estimated strength and the relevant output message.
  */
-Backdrop.evaluatePasswordStrength = function (password, username) {
+Backdrop.evaluatePasswordStrength = function (password, settings) {
   var strength = 0;
   var level = 'empty';
+  var data = settings.data;
+  var config = settings.config;
+  var username = data.username;
+  var email = data.email;
   var hasLowercase = /[a-z]+/.test(password);
   var hasUppercase = /[A-Z]+/.test(password);
   var hasNumbers = /[0-9]+/.test(password);
   var hasPunctuation = /[^a-zA-Z0-9]+/.test(password);
 
-  // If there is a username edit box on the page, compare password to that, otherwise
-  // use value from the database.
+  // If there is a username or email edit box on the page,
+  // compare password to that, otherwise use value from the database.
```

Comment:
Odd text wrapping there:

```suggestion
  // If there is a username or email edit box on the page, compare the password
  // to that; otherwise use the value from the database.
```